.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,3 +26,5 @@ ______
 
 - [ ] Reviewed `Files changed` in the GitHub PR explorer.
 - [ ] Manually tested (in case integration/unit/mock tests are absent).
+- [ ] __If new version release__: Notify the comms team about the pending release and [prepare a message](https://www.notion.so/informalsystems/Communications-pipeline-b8c0eeb71dc24203a048fa6ccf189e1a?pvs=4) to announce it
```

Comment:
It appears to be a link to a restricted page. Should it stay public?

Reply:
it should be a private page, shared with Andrea, then he will manage making the public communications out of that content in that private page.

Reply:
I meant, should the link address be visible to the public - even though not everyone can open it and read its content ? 😄